packages/editor/src/components/editor-container.ts

```diff
@@ -69,6 +90,79 @@ export class EditorContainer
     pageModeSwitched: new Slot(),
   };
 
+  // add files from outside
+  private async _onDropEnd(
```

Comment:
This should be better contained in the `FileDropManager`, the `EditorContainer` should be kept as a thin wrapper.